stable/aws-for-fluent-bit/values.yaml

```diff
@@ -141,6 +164,75 @@ elasticsearch:
   # extraOutputs: |
   #   Index = my-index
 
+s3:
+  enabled: true
+  match: "*"
+  bucket:
+  region: "us-east-1"
+  jsonDateKey: "date"
+  jsonDateFormat: "iso8601"
+  totalFileSize: "100M"
+  uploadChunkSize: 5M
+  uploadTimeout: "10m"
+  storeDir: "/tmp/fluent-bit/s3"
+  storeDirLimitSize: 0
+  s3KeyFormat: /pod-logs/$TAG/%Y/%m/%d/%H/%M/%S
```

Comment:
@PettitWesley

Would it be better for this to be
`/pod-logs/$TAG/%Y-%m-%d/T%H.%M.%S` for shorter file names

Reply:
I don't like dots in S3 keys, but yea good idea, let's split up both by dashes?

@deepend-dev

Reply:
Used this `/pod-logs/$TAG/%Y-%m-%d/%H-%M-%S`